lib/services/menu.service.ts

```diff
@@ -341,7 +350,8 @@ export class MenuService {
           pointer: operationInfo._pointer,
           operationId: operationInfo.operationId,
           operation: operationInfo.operation,
-          deprecated: !!operationInfo.deprecated
+          deprecated: !!operationInfo.deprecated,
+          sortOrder: xSortOrder+""
```

Comment:
Do not transform this to string. This way "11" and "9" will be sorted incorrectly as "9" is greater that "11" in terms of string comparison:

```js
"9".localeCompare("11") === 1
"12".localeCompare("11") === 1
```

as you can see sort order would be broken